streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -947,6 +947,11 @@
         // Private API used to control the emit latency for left/outer join results (https://issues.apache.org/jira/browse/KAFKA-10847)
         public static final String EMIT_INTERVAL_MS_KSTREAMS_OUTER_JOIN_SPURIOUS_RESULTS_FIX = "__emit.interval.ms.kstreams.outer.join.spurious.results.fix__";
 
+        // Private API used to control the usage of consistency offset vectors
+        public static final String IQ_CONSISTENCY_OFFSET_VECTOR_ENABLED = "__iq.consistency.offset"
```

Comment:
Internal feature flag. All work regarding consistency is hidden behind this flag. That's why the `Position` is optional in the stores